charts/seed-bootstrap/charts/elastic-kibana-curator/templates/kibana/kibana-deployment.yaml

```diff
@@ -38,9 +38,20 @@ spec:
         - name: ELASTICSEARCH_URL
           value: http://elasticsearch-logging:{{ .Values.global.elasticsearchPorts.db }}
         ports:
-        - containerPort: {{ .Values.kibanaPort }}
+        - containerPort: {{ .Values.kibana.service.internalPort }}
           name: ui
           protocol: TCP
+{{- if .Values.kibana.readinessProbe.enabled }}
+        readinessProbe:
+          httpGet:
+            path: /status
```

Comment:
While `/status` works fine, I went for `/api/status` as it has a smaller output (not filled with tons of ui and javascript stuff).

Reply:
Thank you for the spot, @dguendisch , its better now.